### ISSUE TITLE
Corrupt data from write via Telegraf procstat plugin.

### ISSUE THREAD

**Issue by nathanielc** [influxdata/influxdb#5662]
I can reliably get Telegraf to write some combination of data that causes this error when attempting to read the data `ERR: invalid character 'j' looking for beginning of value`

Steps:
1. Configure Telegraf with two procstat inputs. As far as I have been able to tell it doesn't matter what the processes are, but that there are two of them. I used this config.
   
   ```
   [[inputs.procstat]]
     exe = "telegraf"
     prefix = "telegraf"
   
   [[inputs.procstat]]
    exe = "kapacitord"
    prefix = "kapacitord"
   ```
   
   where `exe` is the name of an executing binary on your host.
2.  Start InfluxDB/Telegraf and let a least one write happen.
3. Using the telegraf database run the query `select * from procstat`
   You should get the error.

This is likely a bug in Telegraf as well as InfluxDB core.

**Comment by nathanielc** [comment]
@sparrc You might want to take a look at this too.

**Comment by sparrc** [comment]
weird, it seems like it writes correctly at first and then gets corrupted after write ~6?

```
> select * from procstat
name: procstat
--------------
time			cpu_usage		dc		host	memory_rss	memory_swap	memory_vms	pid	process_name
1455315870000000000				us-east-1	tars	17231872	0		2599079936	882	Slack
1455315870000000000				us-east-1	tars	17231872	0		2599079936	589	SpotifyWebHelper
1455315880000000000	0.09995787919900383	us-east-1	tars	17231872	0		2599079936	882	Slack
1455315880000000000	0.09997134762184967	us-east-1	tars	17231872	0		2599079936	589	SpotifyWebHelper
1455315890000000000	0.5000649242287617	us-east-1	tars	17231872	0		2599079936	882	Slack
1455315890000000000	0.500063066603335	us-east-1	tars	17231872	0		2599079936	589	SpotifyWebHelper

> select * from procstat
name: procstat
--------------
time			cpu_usage		dc		host	memory_rss	memory_swap	memory_vms	pid	process_name
1455315870000000000				us-east-1	tars	17231872	0		2599079936	882	Slack
1455315870000000000				us-east-1	tars	17231872	0		2599079936	589	SpotifyWebHelper
1455315880000000000	0.09995787919900383	us-east-1	tars	17231872	0		2599079936	882	Slack
1455315880000000000	0.09997134762184967	us-east-1	tars	17231872	0		2599079936	589	SpotifyWebHelper
1455315890000000000	0.5000649242287617	us-east-1	tars	17231872	0		2599079936	882	Slack
1455315890000000000	0.500063066603335	us-east-1	tars	17231872	0		2599079936	589	SpotifyWebHelper

> select * from procstat
ERR: invalid character 'j' looking for beginning of value
```

**Comment by sparrc** [comment]
btw, it happens for me with just one procstat defined too

**Comment by jwilder** [comment]
Fixed via #5726
